lib/matplotlib/mathtext.py

```diff
@@ -769,7 +769,15 @@ class UnicodeFonts(TruetypeFonts):
 
     def __init__(self, *args, **kwargs):
         # This must come first so the backend's owner is set correctly
-        if rcParams['mathtext.fallback_to_cm']:
+        if rcParams['mathtext.fallback_to_cm'] is not None:
+            if rcParams['mathtext.fallback_to_cm']:
+                rcParams['mathtext.fallback'] = 'cm'
+            else:
+                rcParams['mathtext.fallback'] = None
```

Comment:
When does UnicodeFonts get instantiated?  In general it seems awkward to modify global state at possibly any time.
If the proposal at https://github.com/matplotlib/matplotlib/pull/16509#issuecomment-586646328 makes sense you could just append "cm" to the fallback list when fallback_to_cm is set.

Reply:
Sure this could just be a private var instead